packages/runtime/datastore/src/fluidHandle.ts

```diff
@@ -36,40 +38,40 @@ export class FluidObjectHandle<T extends FluidObject = IFluidObject> implements
         public readonly routeContext: IFluidHandleContext,
     ) {
         this.absolutePath = generateHandleContextPath(path, this.routeContext);
+        if (this.isAttached) {
```

Comment:
If an object is attached (meaning globally attached), it must be locally attached as well.

Reply:
I worry about erosion of meanings. For example, it's not very clear why "isAttached" on routeContext has meaning of globally attached, not locally. Here it does not matter (even if it has different meaning), but in other places it will. I'd try to use different verbs for these things.